### PR TITLE
2802 selfservice portal add support for azure resource groups

### PR DESCRIPTION
**Pull request by HrisyToteva**
closes dfds/cloudplatform/issues/2802

# Additional Review Notes

new functionality for listing and creating azure resources

image

### CODE REVIEW

File: src/src/pages/capabilities/resources/resourceInfoBadges/index.js [hunk 1]
```diff
@@ -186,6 +228,77 @@ export function ResourceInfoBadges() {
...
+                  </Badge>
+                </div>
+              </div>
+            ))
+          ) : (
+            <div>
+              {" "}
+              <strong>
+                <em style={{ color: theme.colors.text.secondary.primary }}>
+                  No Azure resources linked with this Capability.
```
Comment (andreasfrisch) [inline src/src/pages/capabilities/resources/resourceInfoBadges/index.js:258]: I am perfectly okay with this decision.

However, did we consider if we should show a spinner or some other loading indicator instead?
This message might be confusing to users who know they have resources, and the resources simply haven't loaded yet.

The load times might be so small, that this is merely of theoretical concern, so feel free to ignore this :)

File: src/src/pages/capabilities/resources/resourceInfoBadges/index.js [hunk 2]
```diff
@@ -186,6 +228,77 @@ export function ResourceInfoBadges() {
...
+                  <div></div>
+                </>
+              )}
+            </div>
+
+            <Button
+              style={{ marginTop: "1rem" }}
+              onClick={() => handleNewAzureResource()}
+            >
+              {" "}
```
Comment (andreasfrisch) [inline src/src/pages/capabilities/resources/resourceInfoBadges/index.js:296]: Pure curiosity; If we wanted more spacing around the text horizontally, why not use something like `padding: 0 4px;` in the already defined styles?

**Review by andreasfrisch — APPROVED** [review]
Great job! 💯 

I am happy for this to go in so we can get it to people.

I left two comments, but they are not critical -- just some thoughts that we can talk about  in due time.

File: src/src/pages/capabilities/resources/resourceInfoBadges/index.js (hunk 2 above)
Comment (HrisyToteva) [reply src/src/pages/capabilities/resources/resourceInfoBadges/index.js:291]: {" "} this has to be gone

**Pull request merged**